hack/ccm/Dockerfile

```diff
@@ -18,7 +18,7 @@ ARG TARGETPLATFORM=linux/amd64
 ARG ARCH=amd64
 
 # Build IBM cloud controller manager binary
-FROM golang:1.20.12 AS ccm-builder
+FROM golang:1.21.7 AS ccm-builder
```

Comment:
why do you prefer to update ccm, The original ccm still using 1.20. https://github.com/openshift/cloud-provider-powervs/blob/6c98ec5504ba92846fe45b7d1589240208aff6b2/go.mod#L3

Reply:
this version will just use golang 1.21.x to build the image right it will anyways be compatible with 1.20 modules.
I dont see any potential issues here. (we have already updated it in main as well)

Reply:
got it. Thanks